app/apps/workflow/views.py

```diff
@@ -0,0 +1,72 @@
+from apps.users.permissions import rest_permission_classes_for_top
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+from rest_framework import mixins, status, viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from .models import CaseUserTask
+from .serializers import CaseUserTaskListSerializer, CaseUserTaskUpdateOwnerSerializer
+
+role_parameter = OpenApiParameter(
+    name="role",
+    type=OpenApiTypes.STR,
+    location=OpenApiParameter.QUERY,
+    required=False,
+    description="Role",
+)
+
+
+class CaseUserTaskViewSet(
+    mixins.UpdateModelMixin,
+    viewsets.GenericViewSet,
+):
+    permission_classes = rest_permission_classes_for_top()
+    serializer_class = CaseUserTaskListSerializer
+    queryset = CaseUserTask.objects.filter(completed=False)
```

Comment:
`completed=False` could come from the Rest request, similar to the Role filter. That way it's more generic.